containers/Dockerfile.intel-oneapi-hpc-stack-ubuntu20

```diff
@@ -0,0 +1,47 @@
+FROM intel-oneapi-hpckit-dev:ubuntu20
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1
+
+RUN apt-get update -y && \
+apt-get install -y --no-install-recommends -o=Dpkg::Use-Pty=0 \
+        build-essential \
+        ca-certificates \
+        curl \
+        git \
+        gnupg \
+        libexpat1-dev \
+        libssl-dev \
+        m4 \
+        pkg-config \
+        python \
+        vim \
+        wget \
+        --
+
+ENV I_MPI_THREAD_SPLIT=1
+ENV I_MPI_LIBRARY_KIND='release_mt'
+
+RUN cd /tmp && \
+    curl -f -s -S -R -L -O https://github.com/Kitware/CMake/releases/download/v3.16.4/cmake-3.16.4-Linux-x86_64.tar.gz && \
+    mkdir -p /usr/local/cmake && \
+    tar zxf cmake-3.16.4-Linux-x86_64.tar.gz -C /usr/local/cmake --strip-components=1 && \
+    rm cmake-3.16.4-Linux-x86_64.tar.gz && \
+    echo export PATH=/usr/local/cmake/bin:\$PATH >> /etc/bashrc
+
+ENV PNETCDF=/usr/local
+ENV NETCDF=/usr/local
```

Comment:
why are these necessary here?

Reply:
They may not be necessary, but I was having trouble building some of the packages without them.  Perhaps there is a better way to ensure that those environment variables get created and exported along with NETCDF_ROOT/PNETCDF_ROOT.